gengokit/middlewares/templates/endpoints.go

```diff
@@ -5,9 +5,20 @@ package middlewares
 
 import (
 	"github.com/go-kit/kit/endpoint"
+	svc "{{.ImportPath -}} /generated"
 )
 
-func InjectEndpointMiddlewares(in endpoint.Endpoint) endpoint.Endpoint {
+// WrapEndpoint will be called individually for all endpoints defined in
+// the service. Implement this with the middlewares you want applied to
+// every endpoint.
+func WrapEndpoint(in endpoint.Endpoint) endpoint.Endpoint {
```

Comment:
I'm amenable to more considerate names; I think my main change here was for what I thought was a more metaphorically appropriate verb. The issue I have with `WrapAllEndpoints` here is that it works for both functions, depending on whose perspective you view it through. What `WrapAllEndpoints` or `WrapEndpoint`, as it is now, does, in actuality, is wrap a single endpoint. However, the way it is *used* in the template, it ends up wrapping all the endpoints. But this is naming a function for the way it is used, not what it actually does, and trying to reconcile this makes my head fizzy. I'm in favor of more clarifying names, but they cannot be subject to interpretation the way the existing ones seemingly do. My mind fails me at the moment, but I will let it marinate, and likewise, make any other proposals you might come up with.